### PR TITLE
feat: connect state on date dropdown

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] `Error / Warning / Lint / Type`

### Description

SSIA

### Things to Talk About

It looks like using `useProxyValue` with script setup is not working.
Need to research soon.

### CODE REVIEW

File: src/services/dashboards/modules/dashboard-toolset/DashboardToolset.vue [hunk 1]
```diff
@@ -1,16 +1,48 @@
...
+            proxyDateRange: useProxyValue('dateRange', props, emit),
+        });
+
+        return {
+            state,
+        };
+    },
+};
+
+
```
Comment (piggggggggy) [inline src/services/dashboards/modules/dashboard-toolset/DashboardToolset.vue:45]: I also had the same experience. I couldn't use emit in script setup..

**Review by piggggggggy — APPROVED** [review]
LGTM!!

**Review by sulmoJ — APPROVED** [review]
LGTM!!

File: src/services/dashboards/modules/dashboard-toolset/DashboardToolset.vue (hunk 1 above)
Comment (minseolee) [reply src/services/dashboards/modules/dashboard-toolset/DashboardToolset.vue:45]: So sad 😂😂😂

**Pull request merged**